modules/widgets/eu-cookie-law.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * Disable direct access/execution to/of the widget code.
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class Jetpack_EU_Cookie_Law_Widget extends WP_Widget {
+	public static $cookie_name = 'eucookielaw';
+	public static $cookie_validity = 2592000; // 30 days
+
+	public $defaults = array(), $instance;
+
+	function __construct() {
+		parent::__construct(
+			'eu_cookie_law_widget',
+			apply_filters( 'jetpack_widget_name', esc_html__( 'EU Cookie Law Banner', 'jetpack' ) ),
```

Comment:
All code in Jetpack is parsed after each release to build our codex:
http://developer.jetpack.com/

The codex references all actions and hooks in Jetpack, in order to help third-party developers who would like to customize Jetpack.

That means that you'll need to add docblocks for each one of the hooks you add in Jetpack. In this specific case, since the filter isn't new, this should do the trick:
`/** This filter is documented in modules/widgets/facebook-likebox.php */`